src/main.c

```diff
@@ -2757,6 +2844,9 @@ int craft_main(int argc, char **argv) {
         double last_update = glfwGetTime();
         GLuint sky_buffer = gen_sky_buffer();
 
+        bool teleportButtonPressedLastFrame = false;
```

Comment:
This Bool could definitely use a shorter name you use it several times and its a whole line of code.